nvme.c

```diff
@@ -6757,7 +6765,8 @@ static int gen_dhchap_key(int argc, char **argv, struct command *command, struct
 	if (!raw_secret)
 		return -ENOMEM;
 	if (!cfg.secret) {
-		if (getrandom(raw_secret, cfg.key_len, GRND_NONBLOCK) < 0)
+		// if (getrandom(raw_secret, cfg.key_len, GRND_NONBLOCK) < 0)
+		if (syscall(SYS_getrandom, raw_secret, cfg.key_len, GRND_NONBLOCK) < 0)
```

Comment:
Hmm? You first introduce a wrapper, only to call the syscall directly?
Please, no. Use the wrapper.

Reply:
oh... my mistake, Thanks for checking, It didnt saved well